application/controllers/handler.js

```diff
@@ -674,6 +683,7 @@ module.exports = {
               return res(group);
 
             //notify users
+            console.log('Notify '+group.members.length+' users...');
```

Comment:
For debugging =D First you know if the calculation of who should be notified works in general and how much requests should have been send to the activity service.